apps/studio/components/layouts/TableEditorLayout/TableEditorMenu.tsx

```diff
@@ -123,7 +124,7 @@ const TableEditorMenu = () => {
             onSelectSchema={(name: string) => {
               setSearchText('')
               setSelectedSchema(name)
-              router.push(`/project/${project?.ref}/editor`)
+              router.push(`/project/${project?.ref}/editor?schema=${name}`)
```

Comment:
noticed this from https://github.com/supabase/supabase/pull/33122/files - just missing a query param